src/Microsoft.Framework.Runtime/Project.cs

```diff
@@ -188,6 +190,23 @@ internal static Project GetProjectFromStream(Stream stream, string projectName,
                 }
             }
 
+            var fileVersion = Environment.GetEnvironmentVariable("DNX_ASSEMBLY_FILE_VERSION");
+            if (string.IsNullOrWhiteSpace(fileVersion))
+            {
+                project.AssemblyFileVersion = project.Version.Version;
+            }
+            else
+            {
+                try
+                {
+                    project.AssemblyFileVersion = new Version(fileVersion);
+                }
+                catch (FormatException ex)
+                {
+                    throw new FormatException("The assembly file version is invalid: " + fileVersion, ex);
```

Comment:
No strings in the runtime are localizable. I think there is an issue about that somewhere...